### PR TITLE
Migrate to Directory.Build.props

### PR DESCRIPTION
**Pull request by viceroypenguin**

### CODE REVIEW

**Comment by viceroypenguin** [comment]
Optional change - but this is the "expected" way to architect the build system now.

**Comment by MaceWindu** [comment]
examples build broken

File: Tests/FSharp/Tests.FSharp.fsproj [hunk 1]
```diff
@@ -13,7 +12,6 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<Compile Include="AssemblyInfo.fs" />
```
Comment (MaceWindu) [inline Tests/FSharp/Tests.FSharp.fsproj:16]: why it was removed?

Comment (MaceWindu) [reply Tests/FSharp/Tests.FSharp.fsproj:16]: ah I see

**Review by MaceWindu — APPROVED** [review]

**Pull request merged**